Linux/tree-common/init

```diff
@@ -249,6 +249,12 @@ log_begin_msg "Creating default configuration files on rootfs..."
 run mkdir -p "${rootmnt}/etc"
 run touch ${rootmnt}/etc/hostname ${rootmnt}/etc/resolv.conf
 [ -s ${rootmnt}/etc/hostname ] || run sh -ec "echo ${HOSTNAME} > ${rootmnt}/etc/hostname"
+if ! $(cat /etc/hosts | grep -q "127.0.0.1"); then
+echo '127.0.0.1 '${HOSTNAME} >>/etc/hosts
+echo '::1 '${HOSTNAME} >>/etc/hosts
+fi
+sed -i "s/127.0.0.1.*server/127.0.0.1 ${HOSTNAME}/g" /etc/hosts
+sed -i "s/::1.*server/::1 ${HOSTNAME}/g" /etc/hosts
```

Comment:
@meyskens can you prefix the paths with `${rootmnt}`
e.g. : `${rootmnt}/etc/hosts`